garden-backend-service/pyproject.toml

```diff
@@ -40,6 +40,7 @@ pytest-asyncio = "^0.23.7"
 flake8 = "^5.0.4"
 pre-commit = "^3.1.1"
 black = "^24.1.0"
+jupyter = "1.1.1"
```

Comment:
I can remove this if you'd like, but I like having Jupyter around in the dev poetry env to fiddle with code in notebooks